README.md

```diff
@@ -201,17 +201,78 @@ To create a new Breakup Simulation just follow these four steps:
 3. Create your Breakup Simulation
 4. Run and get the Result!
 
-Example:
+### Example CMake integration:
+```cmake
+cmake_minimum_required(VERSION 3.16)
+project(FOO)
 
+# ------------------------- get the breakup model -------------------------
+# Enable ExternalProject CMake module
+include(FetchContent)
+
+# OPTIONAL: disable stuff not needed when used as a lib
+set(BUILD_BREAKUP_MODEL_TESTS OFF CACHE INTERNAL "")
+set(BUILD_BREAKUP_MODEL_SIM OFF CACHE INTERNAL "")
+
+FetchContent_Declare(
+        BreakupModelfetch
+        GIT_REPOSITORY git@github.com:esa/NASA-breakup-model-cpp.git
+        # GIT_TAG master
+)
+
+# Get Breakup Model source and binary directories from CMake project
+FetchContent_MakeAvailable(BreakupModelfetch)
+
+# OPTIONAL: Disable potential warnings
+# from the library target
+target_compile_options(breakupModel_lib PRIVATE -w)
+# from included headers
+get_target_property(propval breakupModel_lib INTERFACE_INCLUDE_DIRECTORIES)
+target_include_directories(breakupModel_lib SYSTEM PUBLIC "${propval}")
+
+# -------------------- define your target and link it ---------------------
+
+add_executable(fooExe foo.cpp)
+target_compile_features(fooExe PUBLIC cxx_std_17)
+target_link_libraries(fooExe PUBLIC breakupModel_lib)
+```
+
+### Example C++ Code integration:
 ```cpp
-    //RuntimeInput via the RuntimeInputSource Object (minmial Config: minL_c = 0.05 + inputSatellites)
+#include <breakupModel/input/RuntimeInputSource.h>
+#include <breakupModel/model/Satellite.h>
+#include <breakupModel/model/SatelliteBuilder.h>
+#include <breakupModel/simulation/BreakupBuilder.h>
+
+int main() {
+    // generate satellites to collide
+    SatelliteBuilder satelliteBuilder;
+    std::vector<Satellite> satellites{
+        satelliteBuilder
+            .setID(0)
+            .setPosition({0.,0.,0.})  // [m]
```

Comment:
WHat happens to this position information? Discarded? Or it gets checked that the two sats are close enough to collide?

Reply:
afaik only the position of the first satellite matters as this is used as the point where the debris is spawned.